### PR TITLE
Conversant: Add consent strings to the user sync pixel

### PR DESCRIPTION
**Pull request by johnwier**
Add consent tags to the Conversant sync endpoint

### CODE REVIEW

File: static/bidder-info/conversant.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&rurl={{.RedirectURL}}"
+    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&rurl={{.RedirectURL}}"
```
Comment (gargcreation1992) [inline static/bidder-info/conversant.yaml:16]: The user sync url works. 👍

File: static/bidder-info/conversant.yaml [hunk 2]
```diff
@@ -13,6 +13,6 @@ capabilities:
...
     userMacro: ""
```
Comment (gargcreation1992) [inline static/bidder-info/conversant.yaml:17]: missing userMacro?

**Review by gargcreation1992 — COMMENTED** [review]
Requesting to please add userMacro.

File: static/bidder-info/conversant.yaml (hunk 2 above)
Comment (johnwier) [reply static/bidder-info/conversant.yaml:17]: it's not missing, we don't use the userMacro.  We append the user id to the end of the redirect url instead.

Comment (gargcreation1992) [reply static/bidder-info/conversant.yaml:17]: Generally it is good practice to replace the macro with UID. Let me know if you plan to make this change.

Comment (johnwier) [reply static/bidder-info/conversant.yaml:17]: I'm not going to make this change right now.  But I'll see how we want to add it in the near future.

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**